service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/AdditionalApplicationsBundle.java

```diff
@@ -66,7 +66,7 @@ public String toLabel() {
             return otherApplicationsBundle.toLabel();
         }
 
-        throw new MissingApplicationException(uploadedDateTime);
+        return otherApplicationsBundle.getUploadedDateTime();
```

Comment:
if there's not an "other" bundle inside the bundle this will cause an NPE I think